internal/cmd/validate.go

```diff
@@ -24,21 +26,28 @@ import (
 )
 
 var (
-	success                = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("10")).Render("Success!")
-	complete               = lipgloss.NewStyle().Bold(true).Render("complete")
-	errorPrefix            = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("9")).Render("error: ")
-	warningPrefix          = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("3")).Render("warning: ")
-	errorMessageStyle      = lipgloss.NewStyle().Bold(true).Width(80)
-	linePrefixStyle        = lipgloss.NewStyle().Foreground(lipgloss.Color("12"))
-	highlightedSourceStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("9"))
-	highlightedLineStyle   = lipgloss.NewStyle().Foreground(lipgloss.Color("9"))
-	codeStyle              = lipgloss.NewStyle().Foreground(lipgloss.Color("8"))
-	highlightedCodeStyle   = lipgloss.NewStyle().Foreground(lipgloss.Color("15"))
-	traceStyle             = lipgloss.NewStyle().Bold(true)
+	// NOTE: these need to be set *after* the renderer has been set, otherwise
+	// the forceColor setting can't work, hence the thunking.
+	success = func() string {
```

Comment:
what's the cost of initializing this each time? alternatively, these could be left as uninitialized vars, and initialize them after we've checked the flag

Reply:
I thought about that, but that spreads the definitions around the file, and some of the definitions are used by multiple functions. I don't think it's particularly expensive, though that isn't because I've profiled it.